tests/pytest/oauth/test_hooks.py

```diff
@@ -50,7 +51,7 @@ def test_pre_logout(app_request, mocked_oauth_analytics_module):
 
     mocked_oauth_analytics_module.started_sign_out.assert_called_once_with(app_request)
     assert not session.logged_in(app_request)
-    assert session.enrollment_token(app_request) is False
```

Comment:
Same note as above, I realized it's not hard to maintain the assertion :relaxed:

Reply:
(see https://github.com/cal-itp/benefits/pull/2910/files#diff-c4e66e5bbdbde74f5062e2a19cb1b0374dc4180b1de7f0b932e169f60ef25e02R54)